server/commands/documentPermanentDeleter.test.js

```diff
@@ -0,0 +1,119 @@
+// @flow
+import { subDays } from "date-fns";
+import { Attachment, Document } from "../models";
+import { buildAttachment, buildDocument } from "../test/factories";
+import { flushdb } from "../test/support";
+import { documentPermanentDeleter } from "./documentPermanentDeleter";
+
+jest.mock("aws-sdk", () => {
+  const mS3 = { deleteObject: jest.fn().mockReturnThis(), promise: jest.fn() };
+  return {
+    S3: jest.fn(() => mS3),
+    Endpoint: jest.fn(),
+  };
+});
+
+beforeEach(() => {
+  flushdb();
+});
```

Comment:
```suggestion
beforeEach(() => flushdb());
```

This is the cause of test failures, need to return the promise

Reply:
Thanks!!